applications/CompressiblePotentialFlowApplication/custom_conditions/incompressible_adjoint_potential_wall_condition.h

```diff
@@ -0,0 +1,179 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//
+//  Main authors:    Marc Nuñez, based on A. Geiser, M. Fusseder, I. Lopez and R. Rossi work
+//
+
+#ifndef KRATOS_INCOMPRESSIBLE_ADJOINT_POTENTIAL_WALL_CONDITION_H
+#define KRATOS_INCOMPRESSIBLE_ADJOINT_POTENTIAL_WALL_CONDITION_H
+
+
+#include "includes/kratos_flags.h"
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/serializer.h"
+#include "includes/condition.h"
+#include "includes/process_info.h"
+#include "potential_wall_condition.h"
```

Comment:
check the includes, same comments as for the element, most of them are alsready included in the baseclass